### ISSUE TITLE
Autocomplete issue with the alias - `ll`

### ISSUE THREAD

**Issue by kiranb83** [warpdotdev/Warp#2949]
### Describe the bug

Autocomplete doesn't seem to work for the `ll` alias.
I don't get autocompletion options and I'm not able to understand why. A couple of aliases,
```
alias ll='ls -lh --color'
alias l='ls -lah'
```

When I type something like `ll /opt/homebrew/`, the only autocomplete option is from history.
OTH, with something like `l /opt/homebrew/`, suggestion for files/folders within /opt/homebrew are shown. This is not specific to that folder, the behaviour is the same for all folders.

### To Reproduce

1. Set up a couple of aliases, e.g.,
```
alias ll='ls -lh --color'
alias l='ls -lah'
```

2. Type in `ll /opt/homebrew/` followed by `Tab` a couple of times.
 - the only autocomplete option I get is from history (last command).
3. OTH, with `l /opt/homebrew/`, suggestion for files/folders within `/opt/homebrew` are shown. This is not specific to that folder, the behaviour is the same for all folders.

### Expected behavior

Autocomplete should be supported in the same way for both aliases.

### Operating System

MacOS

### OS Version

13.3.1

### Shell Version

zsh 5.9 (arm-apple-darwin22.1.0)

### Warp Version

v0.2023.04.11.08.03.stable_00

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by kiranb83** [comment]
I'm not able to upload screenshots. I get an error when I try to (`Something went really wrong, and we can't process that file`).

**Comment by dannyneira** [comment]
Hi @kiranb83 Sorry about this issue. We are tracking this here #2807 

As a workaround I recommend you use ls -hG instead or ls -h --color=auto, you'll get the same effect of adding color, but drop the -- since that appears to be causing some of the issues with the auto-completion.

In general, it appears this issue affects aliases with multiple -'s in combination with '='`s, but we're investigating and will provide updates on the thread above.

Closing as duplicate.

**Comment by kiranb83** [comment]
Thanks for the feedback! I had already tried removing the `--color` option. Didn't change much. Neither did the `-hG` option.

**Comment by dannyneira** [comment]
@kiranb83 it appears that your trying to use TAB to autocomplete, but in Warp since the input editor works like and IDE you need to press ENTER to run the command, TAB opens the completion menu typically.

**Comment by kiranb83** [comment]
@dannyneira I worded my issue poorly, sorry. 
I use ENTER or Ctrl-E for the actual completion. I meant to say that the completion menu hasn't been showing up at all for `ll`. Since it is one of my most frequently used commands, it's been a bit of a pain to have it not working, that's all.

Update from my side. When I changed the alias in my `.zshrc` to not have the `--color` argument, the completion menu appears again. There was no change in behavior when I updated the alias via the `alias` command within my current session.
Sorry about the trouble!
